packages/katello/katello/katello-service

```diff
@@ -76,13 +75,13 @@ def manage_services(action)
     if service_exists?(service)
       case action
       when 'enable', 'disable'
-        puts "systemctl #{action} #{service}.service"
-        puts `systemctl #{action} #{service}.service`
-      else
-        puts "#{COMMAND} #{service} #{action}"
-        puts `#{COMMAND} #{service} #{action}`
+      puts "systemctl #{action} #{service}.service"
+      puts `systemctl #{action} #{service}.service`
+    else
+      puts "#{COMMAND} #{service} #{action}"
```

Comment:
This needs to be the other way around because systemctl has a different argument order. This also means the whole case statement can be simplified because there's no difference between the actions.

Reply:
you can also DRY it up:
```
sys_cmd = "#{COMMAND} #{service} #{action}"
`#{sys_cmd}`
```